media_kit_video/lib/src/video/video_web.dart

```diff
@@ -450,7 +452,7 @@ class VideoState extends State<Video> with WidgetsBindingObserver {
 /// Makes the native window enter fullscreen.
 Future<void> defaultEnterNativeFullscreen() async {
   try {
-    await document.documentElement?.requestFullscreen();
+    await document.documentElement?.requestFullscreen().toDart;
```

Comment:
Does this work?

Reply:
just tested and yes it works on javascript code (cant compile to wasm since the player is not migrated yet but if it works for javascript it should work for wasm)